learning-bitmark/working-with-bitmarks/using-bitmark-shares.md

```diff
@@ -0,0 +1,631 @@
+
+*Example:*
+
+* Run bitcoind
+
+  ```shell
+    $ bitcoind -datadir=${HOME}/.config/bitcoin/
+  ```
+
+<br>
+
+* Pay for the transaction by bitcoin using the Bitmark-Wallet
+
+  ```shell
+    $ bitmark-wallet --conf ${XDG_CONFIG_HOME}/bitmark-wallet/test/test-bitmark-wallet.conf \
+      btc --testnet \
+      sendmany --hex-data \
+      'b5000d490f2194307db84d86c6bed022cadf1c6f5cef7e5eb73ac871454aa5bf8ae3219cfe73fd6c53770f7ecba7b413' \
+      '2MvrbNpfr6zWG2Mm2hnCh735hPwjoist3Ep,20000'
+  ```
+  ```json
+    {
+      "txId": "fc208c0...12c3d82",
+      "rawTx": "0100000001e45...7b41300000000"
+    }
+  ```
+
+<br>
+
+### Verifying the status of the share creation transaction
+
+  ```shell
+    $ bitmark-cli -n <network> \
+      status -t <txid>
+  ```
+
+  >The `status` command is to query a transaction's status. The options:
+  >
+  >* `txid` - The transaction id which is printed in the output of the share creation command.
+  >
+  >The returned status:
+  >
+  >* `Pending` - Has not been paid.
+  >
+  >* `Verified` - Paid but not confirmed on the blockchain.
+  >
+  >* `Confirmed` - Confirmed on the blockchain.
+
+
+*Example:*
+
+  ```shell
+    $ bitmark-cli -n testing \
+      status -t \
+      6f8cccbb8db2e2304227abb91cf0cbceaed79376b9d7b0181d28fa20f04fe2a3
+  ```
+  ```json
+    //Right after the payment
+    {
+      "status": "Verified"
+    }
+
+    //Check again fter some minutes
+    {
+      "status": "Confirmed"
+    }
+  ```
+
+<br>
+
+### Verifying that the share has been created and allocated to the owner
+
+  ```shell
+    $ bitmark-cli -n <network> \
+      balance -o <ownwer identity>
+
+  ```
+
+  >The `owned` command is to list out all the bitmarks and shares are being owned by a user. The options:
+  >
+  >* `owner identity` - The identity of the owner of the bitmark from which the shares are created.
+
+
+*Example:*
+
+  ```shell
+    $ bitmark-cli -n testing \
+      balance -o first
+  ```
+  ```json
+    {
+      "balances": [
+        {
+          "shareId": "b069f2956b828281dec040782eea3d63793ab4cf17c26f7639e95f6f3b20ba23",
+          "confirmed": 1000,
+          "spend": 0,
+          "available": 1000
+        }
+      ]
+    }
+  ```
+
+<br>
+<br>
+
+## Granting bitmark shares to another account
+
+Any owner with a non-zero share balance can grant shares from that balance to another account.
+
+A grant share transaction is a two signature transaction. Therefore, the current share owner initializes the transaction and the new owner needs to sign to accept the shares.
+
+  ```shell
+    # The current owner initializes the grant shares transaction
+    $ bitmark-cli -n <network> -i <current owner identity> \
+      grant -r <receiver> -s <shareid> -q <quantity>
+
+    # The receiver signs to accept the shares
+    $ bitmark-cli -n <network> -i <receiver identity> \
+      countersign -t <hex-data>
+  ```
+
```

Comment:
I found this is a little muddy, listing these both together, then trying to explain the variables all at once. Putting each in its own subsection as I suggest above would help clarify that.

Reply:
Updated.